Fix verify job (#31)
ebi-ait/ingest-kube-deployment: infra/helm-charts/mongo/charts/ingestbackup/README.md

### Quickstart

> Ingest Backup is deployed through Helm on Kubernetes through the `setup.sh` script provided in this module. The setup script [requires AWS access keys and role ARN](#credentials) can be downloaded through the AWS console in CSV format. With the addition of the [automated verification procedure](#verification), the setup script also requires the Slack Webhook URL where alerts can be sent. This can be retrieved through Slack apps management on their Web site.

Reviewer: Is this not relevant anymore?

Author: No, this setup script is no longer needed and redundant with the mongo helm chart setup.sh. This is not the proper way to deploy the backup and verify cronjobs.

    SLACK_WEBHOOK=http://slack/webhook/url \
    docker-compose -f compose-verify.yml up --build &&\
    docker-compose -f compose-verify.yml rm -fsv

### Known warning in ingest-backup-verify logs

> The ingest-backup-verify job logs have the following failure messages in the logs. These warnings are as designed in mongo, see

Reviewer: ```suggestion
The ingest-backup-verify job logs have the following known failure messages in the logs. See MongoDB's documentation related to this:
```

https://jira.mongodb.org/browse/SERVER-39390.

```
2021-09-29T15:52:52.560+0000 I  NETWORK  [js] DBClientConnection failed to receive message from localhost:27017 - HostUnreachable: Connection closed by peer
server should be down...
2021-09-29T15:52:52.578+0000 I  NETWORK  [js] trying reconnect to localhost:27017 failed
2021-09-29T15:52:52.578+0000 I  NETWORK  [js] reconnect localhost:27017 failed failed
2021-09-29T15:52:52.579+0000 I  QUERY    [js] Failed to end session { id: UUID("29049dba-a2f2-473c-bc7f-c8dd2b3cb029") } due to SocketException: socket exception [CONNECT_ERROR] server [couldn't connect to server localhost:27017, connection attempt failed: SocketException: Error connecting to localhost:27017 (127.0.0.1:27017) :: caused by :: Connection refused]
```

### Manually triggering a job from cronjob

Command:
```
   kubectl create job --from=cronjob/<name of cronjob> <name of job>
```

To manually trigger a backup job:
```
   kubectl create job --from=cronjob/ingest-backup ingest-backup-manual-datetime
```

To manually trigger a verify job:
```

> kubectl create job --from=cronjob/ingest-backup-verify ingest-backup-verify-manual-datetime

Reviewer: Just to use the same formatting as above:

```suggestion
   kubectl create job --from=cronjob/ingest-backup-verify ingest-backup-verify-manual-datetime
```
